src/SQLChecks/Tests/SQLAgent.tests.ps1

```diff
@@ -20,4 +20,10 @@ Describe "SQL Agent status" -Tag AgentIsRunning {
     It "SQL Agent is running on $serverInstance" {
         (Get-SqlAgentService -Config $Config).Status | Should Be "Running"
     }
+}
+
+Describe "Unconfigured SQL Agent Alerts" -Tag UnconfiguredSQLAgentAlerts {
```

Comment:
This tag should match the documentation and function - `CheckUnconfiguredSQLAgentAlerts`